COLLADAStreamWriter/include/COLLADASWConstants.h

```diff
@@ -202,6 +202,8 @@ namespace COLLADASW
         static const String CSW_ELEMENT_PASS;
         static const String CSW_ELEMENT_PH;
         static const String CSW_ELEMENT_PHONG;
+		static const String CSW_ELEMENT_PLUGIN_VERSION;
```

Comment:
I'm a bit confused, could you help me understand:

COLLADAMayaFileTranslator.cpp , line 85

this creates a ```revision``` string that contains ```TRANSLATOR_VERSION + CURRENT_REVISION + COLLADAMaya::CURRENT_SHA1```   .  Where is this string saved in the .dae ?

COLLADAMayaDocumentExporter.cpp line 382
This adds CURRENT_REVISION and  CURRENT_SHA1 to the <COLLADA><asset> tag (as extra?)   

thanks!